backend/controllers/block-controller.mjs

```diff
@@ -0,0 +1,57 @@
+import Block from '../classes/Block.mjs';
+
+// Dummy data for testing...
+const blockchain = { chain: [Block.genesis] };
```

Comment:
You can discard this when merging to production.

You can still "imagine" the presence of a `blockchain` variable that will be imported into the controller from another module, potentially from `app.mjs`.